WF_Proton_Launcher/Main.cpp

```diff
@@ -142,6 +146,8 @@ int main(int argc, char** argv)
 				<< "  -v  Print more verbose output" << std::endl
 				<< "  -32 Launch the 32-bit binary instead of the 64-bit one" << std::endl
 				<< "  -fullscreen Launch the game in full-screen" << std::endl
+				<< "  -language:xx changes the game's language. example: -language:en" << std::endl
```

Comment:
Nit; I'd change "example:" to instead say "Default:", that clarifies both as an example and as the default value.

Reply:
Alright !